src/commands/init.js

```diff
@@ -200,7 +200,7 @@ class InitCommand extends Command {
   }
 }
 
-InitCommand.description = `Configure continuous deployment for a new or existing site`
+InitCommand.description = `Configure continuous deployment for an existing site or create a new site without continuous deployment`
```

Comment:
We might need to clarify this.
The `ntl init` command lets you:
1. Configure continuous deployment for an existing site
2. Create a new site with continuous deployment
3. Create a new site without continuous deployment

Creating a new site without continuous deployment is more of a fallback when the project is not configured as a `git` repo.
If users only want to create a site we can direct them to `ntl sites:create`

Reply:
Hmm. Do we want to not mention "Create a new site without continuous deployment" specifically then? Instead, pointing people to `ntl sites:create`?

For example: "Configure continuous deployment for a new or existing site. To create a new site without continuous deployment, use [`ntl sites:create`](https://cli.netlify.com/commands/sites#sitescreate)"

Reply:
> For example: "Configure continuous deployment for a new or existing site. To create a new site without continuous deployment, use [`ntl sites:create`](https://cli.netlify.com/commands/sites#sitescreate)"

Yes! I think this is preferable if the user's intention is to solely create a new site.

Also, for the description we should use `netlify sites:create` instead of `ntl sites:create` for consistency with the other docs.